eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -0,0 +1,8 @@
+<!-- When altering this file or making other Source Build related changes, include @dotnet/source-build as a reviewer. -->
+<!-- See aka.ms/dotnet/prebuilts for guidance on what pre-builts are and how to eliminate them. -->
+
+<UsageData>
+  <IgnorePatterns>
+    <UsagePattern IdentityGlob="Polyfill/*" />
```

Comment:
The reference to Polyfill will either need to be removed/conditionally excluded from source-build else it will also need to be added to source-build-externals as well.

Reply:
I tried to exclude but we rely on it in code, it does not just polyfill for older tfms. What does it take to get it into source-build-externals?

Can you also please advise on how to get rid of these prebuilts? 

![image](https://github.com/user-attachments/assets/c67bf5da-8940-4135-b5fe-54d3c9a78f0e)

Reply:
How is polyfill used in the product?  I'm am just trying to make sure it is really needed for source-build.  The instructions for adding new source-build-externals is at https://github.com/dotnet/source-build-externals/blob/main/README.md#adding-a-new-external-component.  You will want to explore the dependencies Polyfill has to ensure it is source build compatible (at least the components .NET would have a dependency on).

Reply:
Your prebuilts appear to be arcade related.  Why is the version.details arcade dependency pointing to 10.0 while the global.json is downgrading to a 9.0 version.  I would expect 10 to be used in both.  What version of the SDK will these changes be flowing into?

Reply:
These are the versions we are using in vstest and there we don't have these problems. Not sure what is missing here.

Reply:
Polyfill does not have any deps, it also does not add any dll. It just brings code as nuget package so the same modern C# syntax can be used in older tfms.